telemetry/specs/INT.mdk

```diff
@@ -1538,6 +1750,42 @@ the GRE was inserted by the INT source.
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
+
```

Comment:
IMO it would be cleaner to add a section header. Change the title of the previous example, replacing "INT" with "INT-MD", then add a title here with "INT-MX".

Reply:
Done.